### ISSUE TITLE
Issue with running startStandalone()

### ISSUE THREAD

**Issue by gust42** [locize/locize#171]
## 🐛 Bug Report
when running startStandalone() in an react-intl project
i get the following error:
implementation.bindLanguageChange is not a function

this is because startStandalone.js is missing bindLanguageChange and bindMissingKeyHandler in its startLegacy call

```
export function startStandalone () {
  startLegacy({
    getLocizeDetails: () => {
      return {}
    },
    getLng: () => {
      return undefined
    },
    setResource: () => {},
    triggerRerender: () => {},
    getResourceBundle: () => {
      return {}
    }
  })
}
```
to fix it one could add ?. on the function calling in processLegacy.js but i dont know what other effects that would have

```
  // start stuff
  implementation?.bindLanguageChange(lng => {
    api.sendCurrentTargetLanguage(implementation.getLng())
  })
  implementation?.bindMissingKeyHandler((lng, ns, k, val) => {
    api.onAddedKey(lng, ns, k, val)
  })
```

## To Reproduce

Start a new project with react-intl and try to call startStandalone()

## Expected behavior

It should loads for using incontexteditor

## Your Environment

nextjs
locize 3.22
react-intl

**Comment by jamuhl** [comment]
should be fixed in locize@3.2.3

**Comment by gust42** [comment]
It works, thanks for a fast response
